### PR TITLE
Add community signing pipeline

### PR DESCRIPTION
**Pull request by Allda**
The new pipeline is added to Ansible configuration. The pipeline
triggers a index signing process and is requested by operator community
project. The pipeline is protected by NGINX proxy with basic auth. Only
selected usernames can trigger the pipeline.

JIRA: ISV-1485

### CODE REVIEW

**Comment by Allda** [comment]
@amisstea @wcheang Please review this WiP PR. I am currently waiting for all the tekton tasks to be ready so I can include them in the pipeline, but there is already a basic skeleton of new pipelines + ansible configuration.

File: ansible/roles/operator-pipeline/tasks/webhook-community-signing-event-listener.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          metadata:
+            name: community-signing-pipeline-listener
+            labels:
+              app: community-signing-pipeline
+              suffix: "{{ suffix }}"
+              env: "{{ env }}"
+          spec:
+            serviceAccountName: pipeline
+            triggers:
+              # run Hosted pipeline on PR opened, reopened, synchronized
```
Comment (wcheang) [inline ansible/roles/operator-pipeline/tasks/webhook-community-signing-event-listener.yml:23]:
```suggestion
              # run community signing pipeline on PR opened, reopened, synchronized
```

File: ansible/roles/operator-pipeline/tasks/webhook-community-signing-trigger.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      k8s:
+        state: present
+        namespace: "{{ oc_namespace }}"
+        definition:
+          apiVersion: triggers.tekton.dev/v1alpha1
+          kind: TriggerTemplate
+          metadata:
+            name: operator-hosted-pipeline-trigger-template
+            labels:
+              app: operator-pipeline
```
Comment (wcheang) [inline ansible/roles/operator-pipeline/tasks/webhook-community-signing-trigger.yml:35]: Should this still be operator-pipeline?

**Comment by wcheang** [comment]
Looks okay so far. Although I'm not familiar with nginx, so I can't say I would be able to spot anything wrong with the nginx related configs...

File: ansible/roles/operator-pipeline/tasks/webhook-community-signing-trigger.yml (hunk 2 above)
Comment (Allda) [reply ansible/roles/operator-pipeline/tasks/webhook-community-signing-trigger.yml]: Fixed

File: ansible/roles/operator-pipeline/tasks/webhook-community-signing-event-listener.yml (hunk 1 above)
Comment (Allda) [reply ansible/roles/operator-pipeline/tasks/webhook-community-signing-event-listener.yml]: Fixed

**Comment by Allda** [comment]
@wcheang @amisstea The new signing pipeline is now finished. Please give it a final review. Thanks

**Review by wcheang — APPROVED** [review]
LGTM :+1:

**Pull request merged**